agent/proto/agentpb/agent.proto

```diff
@@ -101,6 +101,9 @@ message Probe {
     // Warning defines a (possibly transient) condition that requires attention
     // but is not critical
     Warning = 2;
+    // Info defines an informational probe condition that does not
+    // lead to the degraded state
+    Info = 3;
```

Comment:
Could be, but warnings seem to be used everywhere now and lead to the degraded status. I don't see why this is bad though, to me it's reasonable: 

* `critical` means the problem is "very bad" and requires immediate attention and leads to degraded status. Is it actually used anywhere?
* `warning` means the problem is not as critical but cluster is still degraded.
* `info` means there is something wrong but does not lead to degradation in service.

Reply:
I just looked at the uses of `Probe_Warning` and only [one](https://github.com/gravitational/satellite/blob/6.0.2/monitoring/timedrift.go#L246) stood out coupled with `Probe_Failed` status as wrong to me - this should be `critical` instead. I think `warning` is precisely what you describe for `info` - a hint at some condition which does not degrade the cluster. `Info` is what we pass on as running probes.

Reply:
my 2 cents, if I were coming across this, to me I interpret as:
critical - things are on fire and something isn't working (degraded)
warning - things could potentially be on fire but are probably working. Changes to the cluster are unsafe (degraded)
info - something an admin should be aware of, but it's not clearly breaking the cluster.

Although I suspect the things we really want to report will be warnings or critical, I think I sort of agree that while we don't have a definition for certain labels that informational is appropriate (or maybe better to mark unknown since we haven't set a severity).